wiki/Help_Center/en.md

```diff
@@ -237,17 +221,15 @@ If smoke appears when you don't want it to, go to **Main Menu**, from there sele
 
 ------------------------------------------------------------------------
 
-How do I save my own replay of a completed song?
-------------------------------------------------
+### How do I save my own replay of a completed song?
 
 You can save your replay after you have completed the song, in order to do that, once you are at the post-game results screen, hit the **F2** button. That will save the replay in the **/Replays/** folder under the osu! installation directory.
```

Comment:
Change `**F2**` to `` `F2` ``
Change `**/Replays/** to `` `Replays` `` (I think we have decided to use code blocks for folder names and directories